chapters/fa/glossary/1.mdx

```diff
@@ -73,68 +74,6 @@
 | Self-Contained        | خودکفا           |
 | Script                | اسکریپت‌          |
 | Folder | پوشه |
-| Neural Network | شبکه عصبی |
```

Comment:
I think there's been a mistake. If this goes through we will be removing 61 words from the glossary! That is like, weeks of work. Please only add the ones for this PR.